src/lab/Graphs/LineAreaGraph/LineAreaGraph.stories.mdx

```diff
@@ -28,6 +28,7 @@ in form of lines, area-closed under the graph and events.
       yLabelOffset: 35,
       marginLeftEventTable: 10,
       widthPercentageEventTable: 40,
+      xAxistimeFormat: "hh:mm:ss Do MMM",
```

Comment:
Do or DD? @whitetiger1399

Reply:
`Do` is for day of the month with ordinal i.e. `1st 2nd ...31st` and `DD` is for day of the month in 2-digits `01-31`

Reply:
Looks good then.